source/stylesheets/_tab-code_signing.scss.erb

```diff
@@ -258,34 +259,48 @@
 
 	section.code-signing-identities {
 
+		header {
+
+			.title {
+				width: 35%;
+			}
+
+			.password {
+				width: 31%;
+				margin-bottom: 0;
+			}
+		}
+
 		ul.uploaded-files {
 
 			li.uploaded-file {
 
-				&:before {
-					background-image: image-url("code_signing/file-purple.svg");
-				}
-
 				.details {
-					@include flex-grow(1);
-					width: 40%;
+					width: 35%;
+
+					&:before {
+						background-image: image-url("code_signing/file-purple.svg");
+					}
 				}
 
 				.password-edit {
-					@include flex-grow(1);
-					@include flex-shrink(0);
 					@include display-flex;
 					@include align-items(center);
-					margin-left: 6px;
-					margin-right: auto;
+					width: 31%;
 
 					form.password {
 						@include display-flex;
 
+						.input-with-show-password {
+							position: relative;
+							@include display-flex;
+						}
+
 						input:not([type="submit"]) {
```

Comment:
This will also select the eye checkbox. You should add a class to the input field (like `input.password`, and modify this selector to use that, and not the `:not(...)`.